purchase_order_return/models/purchase_order_line.py

```diff
@@ -24,7 +24,7 @@ def onchange_product_id(self):
     @api.model
     def create(self, values):
         line = super().create(values)
-        if "return_qty" in values:
+        if "return_qty" in values and values.get("return_qty", False):
```

Comment:
Si values no tiene "return_qty", values.get("return_qty", False) devuelve FALSO, no necesita poner "return_qty" in values

Reply:
Bale, zuzenduta ta ulertuta.